src/client/public/index.script.ts

```diff
@@ -11,6 +13,31 @@ import { Strings } from '../app/strings';
     document.getElementById('choose-your-host').textContent = strings.HtmlPageStrings.chooseYourHost;
     document.getElementById('choose-your-host').style.visibility = 'visible';
 
+    let ribbons = document.getElementsByClassName('ribbon');
+    for (let i = 0; i < ribbons.length; i++) {
```

Comment:
Dont do `ribbons.length`. Access the `length` property is more complex. Given modern machines you'd still be able to getaway using `ribbons.forEach(ribbon => {})`. If you absolutely need max perf, then go ahead and do:

`for (const ribbon of ribbons) { }`